src/elements/content-sidebar/activity-feed/annotation-thread/__tests__/AnnotationThreadCreate.test.js

```diff
@@ -93,6 +107,24 @@ describe('elements/content-sidebar/activity-feed/annotation-thread/AnnotationThr
         expect(getByTestId('annotation-create')).toHaveClass('is-pending');
     });
 
+    test('should use annotatorEvents on annotation add', () => {
+        mockCreateAnnotation = jest.fn((fileId, fileVersionId, payload, filePermissions, successCallback) => {
+            successCallback(annotation);
+        });
+        const props = getDefaultProps();
+        const { getByText } = getWrapper(props);
+        const { eventEmitter } = props;
+
+        fireEvent.click(getByText('Post'));
+
+        expect(useAnnotatorEvents).toHaveBeenCalledWith({ eventEmitter });
```

Comment:
In BUIE it's common to use `ToBeCalledWith` instead of `toHaveBeenCalledWith`.
Also applies to other use cases below and in `useAnnotationThread.test.js`